nservicebus/hosting/handling-failures.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Handling failures 
+summary: How to subscribe to notifications regarding critical errors which adversely affect messaging in your endpoint.
+tags:
+- NServiceBus Host
+- Self Hosting
+---
+
+NServiceBus offers an error event handler that you can use in order to receive notifications when something undesirable happens in the endpoint which involves the ability to receive or send messages. E.g. When your persistence storage that is essential for messaging goes down. 

```

Comment:
>something undesirable happens

should it be "critical error takes place" to signify importance of this notifications?